### ISSUE TITLE
aria-describedby for transcripts for audio-only? (WCAG SC 1.2.1)

### ISSUE THREAD

**Issue by medeaoblongata** [w3c/wai-media-guide#164]
Very surprised that aria-describedby is not mentioned as a viable mechanism for programmatically associating audio-only with a transcript on the same page. 

It's not mentioned as a sufficient technique in Understanding SC 1.2.1 either.

Is there a reason for this omission? Is it the wrong attribute to use? Is this approach deprecated?

**Comment by medeaoblongata** [comment]
OK, I was provided with some answers on the (excellent) web-a11y slack channels. Briefly:

- aria-describedby is just a flat text, with no way to navigate or browse through it. Especially with long transcripts it does not make a lot of sense to use that mechanism. 
- If aria-describedby is placed an audio element, only AT users would benefit from this, and of these a large number of users will already be able to hear the audio.
- Levels of support for aria-describedby on an audio element within ATs are unknown and possibly poor.

These three points are so compelling that I would like to request that caveats covering the limitations of aria-describedby appear on the wai-media-guide page about transcripts.

**Comment by shawna-slh** [comment]
@brennanyoung Thank you much for raising this issue, for following up to get additional perspectives, and for updating the issu. with these points. I appreciate your time and input.

EOWG would like to keep this resources as succinct as feasible - and focus on what to do.

I think it would add a fair bit of complexity to add these reason why not do use aria-describedby -- especially for readers not familiar with ARIA. And it seems not worth the added info to say what not to do, instead of what to do.

Please feel free to re-open this issue if you would like EOWG to discuss it.
